src/slf4ec.c

```diff
@@ -166,68 +168,26 @@ LogResult noLog()
     return returnCode;
 }
 
-/**
- * Macro function to avoid duplicate code inside nfLog1() and yfLog1().
- */
-#define HANDLE_LOG1(mFile, mLine, mFunction)                                            \
```

Comment:
Special macro code has been moved into the _privatelog() function to clean up the code and to allow debuggers to step through that code.